tests/test_datacite.py

```diff
@@ -1,22 +1,119 @@
-from transmogrifier.helpers import parse_xml_records
-from transmogrifier.sources.datacite import Datacite
-
+from pytest import raises
 
-# TODO: add tests for expected data issues:
-# missing required Datacite fields raises error
-# all fields transformed successfully
-# repeatable fields transformed successfully
+from transmogrifier.models import Contributor, Date, Identifier, Note, TimdexRecord
+from transmogrifier.sources.datacite import Datacite
 
 
-def test_jpal_fixtures():
-    input_records = parse_xml_records("tests/fixtures/datacite_jpal_records.xml")
+def test_jpal_fixtures(datacite_jpal_records):
```

Comment:
This was my bad but we should name this better, something along the lines of `test_datacite_iterates_through_all_records`.